bravado/mapping/spec.py

```diff
@@ -38,17 +56,47 @@ def build(self):
         validator20.validate_spec(self.spec_dict)
```

Comment:
as we have the version differentiator in place in ssv, we can call the generic validating method.